css/font-awesome.css

```diff
@@ -0,0 +1,3795 @@
+@font-face {
```

Comment:
I know this file gets built when you run `make watch` or `make css`, but I don't think it should be included... I should probably update the Makefile to avoid building it.

Please remove it from this pull request.

Reply:
Ok, will do.